### ISSUE TITLE
(temporarily) disable device (and filter those)

### ISSUE THREAD

**Issue by HA-TB303** [esphome/feature-requests#2170]
**Describe the problem you have/What new integration you would like**

- The ability to temporarily disable a device which is offline.
- Disabled devices are excluded from "update all"
- Disabled devices can be visually filtered from the list (exactly like in Home Assistant device listing)

**Please describe your use case for this integration and alternatives you've tried:**

**Additional context**

**Comment by Nosen92** [comment]
I very much welcome this feature. Some YAML configs take quite some effort to write, so the user might resist deleting them (understandably). Being able to disable and archive configs would allow you to experiment without cluttering the dashboard. It would also make it easier to return to an esphome project.

**Comment by kirill-sh** [comment]
I with you. This is a much needed feature.

**Comment by walzing** [comment]
Same here - I have now 10 Plug showing offline

**Comment by jath58** [comment]
Same here, much needed when devices are temporarily not used (but will come back).

**Comment by tevaadarelsol** [comment]
+1 for this feature request...

**Comment by brotaxt** [comment]
+1 :)

**Comment by WagnerSystems** [comment]
+1

**Comment by larsoss** [comment]
+2 ;-)

**Comment by vb7445** [comment]
+3 :)

**Comment by Sansei-Innovation** [comment]
Agree.  Alternatively, the ability to organize devices in (user defined nameed) folders.  This would enable the creation of "Disabled" or "Archive" folders.  Under this scenario, "_Update All_" could either be constrained to the active folder or a pop could toggle "Active Folder" or "All Folders"

**Comment by broyuken** [comment]
I would like this as well, I have some devices that I like to tinker with when I have time but are a WIP, and so those are in my esphome dashboard, but a lot of the time they are in a drawer not connected, so I would like to

1. be able to filter them out so they don't show with my active/connected devices in the dashboard
2. don't try to be updated when I click update all in the dashboard

**Comment by qweritos** [comment]
+1
